File: script/ruby/println/fizz_buzz_println.rb
# frozen_string_literal: true

n = ARGV[0].to_i

# start_time = Time.now
Range.new(1, n).each do |i|
  if (i % 15).zero?
    print "fizzbuzz\n"
  elsif (i % 3).zero?
    print "fizz\n"
  elsif (i % 5).zero?
    print "buzz\n"
  else
    print "#{i}\n"
  end
end

# p "elapsed_time:#{Time.now - start_time}[sec]"

File: script/python/println/fizz_buzz_println.py
import sys
# import time

if __name__ == '__main__':
    n = int(sys.argv[1])

    # start = time.time()
    for i in range(1, n + 1):
        if i % 15 == 0:
            print('fizzbuzz')
        elif i % 3 == 0:
            print('fizz')
        elif i % 5 == 0:
            print('buzz')
        else:
            print(i)
    # print("elapsed_time:{0}".format(time.time()) + "[sec]")
